go/vt/sqlparser/token.go

```diff
@@ -34,22 +34,23 @@ const (
 // Tokenizer is the struct used to generate SQL
 // tokens for the parser.
 type Tokenizer struct {
-	InStream            io.Reader
-	AllowComments       bool
-	SkipSpecialComments bool
-	SkipToEnd           bool
-	lastChar            uint16
-	Position            int
-	OldPosition         int
-	lastToken           []byte
-	lastNonNilToken     []byte
-	LastError           error
-	posVarIndex         int
-	ParseTree           Statement
-	partialDDL          *DDL
-	nesting             int
-	multi               bool
-	specialComment      *Tokenizer
+	InStream             io.Reader
+	AllowComments        bool
+	SkipSpecialComments  bool
+	SkipToEnd            bool
+	lastChar             uint16
+	Position             int
+	OldPosition          int
+	lastToken            []byte
+	lastNonNilToken      []byte
+	LastError            error
+	posVarIndex          int
+	ParseTree            Statement
+	partialDDL           *DDL
+	nesting              int
+	multi                bool
+	specialComment       *Tokenizer
+	potentialAccountName bool
```

Comment:
I don't love this method of state tracking, try to come up with something better as you keep working on this